home.html

```diff
@@ -307,21 +307,54 @@ <h2 class="sub-title mt-2 d-flex gap-3 mb-3">
                             <strong> Meaningful-image-name</strong> All
                             meaningful images that are presented to the user
                             have equivalent, accessible descriptions.
+                            <a
```

Comment:
Please set the correct link with github use this 
https://mubeenfayyaz-arbisoft.github.io/Web-Accessibility/meaningful-image-name.html 
Instead of just file name same for other bottom links